pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -24,6 +23,13 @@ import (
 	"github.com/openshift/origin/pkg/client"
 	imageapi "github.com/openshift/origin/pkg/image/api"
 	"github.com/openshift/origin/pkg/image/importer"
+	quotautil "github.com/openshift/origin/pkg/quota/util"
+)
+
+const (
+	// DockerRegistryURLEnvVar is a mandatory environment variable name specifying external url of internal
+	// docker registry. All references to pushed images will be prefixed with its value.
+	DockerRegistryURLEnvVar = "DOCKER_REGISTRY_URL"
```

Comment:
This PR doesn't change the semantics of `DOCKER_REGISTRY_URL`. It just exports the variable and adds a documentation of the current state.

The comment is flawed however. The url is internal and possibly external as well, so I'll ditch the `external` qualifier.